core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -61,6 +61,7 @@
   @Deprecated // to be removed before 2.0
   SqlConformanceEnum PRAGMATIC_2003 = SqlConformanceEnum.PRAGMATIC_2003;
 
+  SqlConformanceEnum HIVE = SqlConformanceEnum.HIVE;
```

Comment:
Out of curiosity: other SqlConformanceEnum are all marked as `@Deprecated`. Does the new `HIVE` should be marked as the same (there might be a context that all SqlConformanceEnum will be deprecated somehow)?

Reply:
In fact I think HIVE here has to be marked as deprecated.

The long-term-support version is defined in `SqlConformanceEnum.java`

Reply:
You're right. I should put it SqlConformanceEnum.java. Thank you for your kindly reminder.